sql/moz-fx-data-shared-prod/braze_derived/suppressions_v1/metadata.yaml

```diff
@@ -0,0 +1,17 @@
+friendly_name: Suppressions
+description: |-
+  Suppression Data from Acoustic and Campaign Monitor. This also includes users
```

Comment:
I think we might want to just say this is the suppression list based on Acoustic and CTMS. Anyone from Campaign Monitor's suppression list that happens to be included is _not_ included based on the SQL logic in the data model, but based on Alexander manually syncing them into Acoustic 😓 I want to make sure we don't confuse ourselves down the line that this model is somehow accessing data from Campaign Monitor.